tests/bidiSupport/numericShaping/runTests.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
This file really doesn't belong. We shouldn't be using both DOH and Intern to test this, so this should be removed.